packages/frontend/src/components/authenticate.tsx

```diff
@@ -81,13 +81,15 @@ export const Authenticate = ({ onCancel }: AuthenticateProps) => {
                     <>
                         <LogoIcon className="w-40 h-40" />
                         <Typography
+                            data-testid="welcome-to-carrot-modal-title"
```

Comment:
This id should follow the convention of `component_name-...`, so for example here it could be `authenticate-modal-title`.